verification/verify.go

```diff
@@ -153,3 +163,68 @@ func IsRetryable(err error) bool {
 	_, retryable := err.(Retryable)
 	return retryable
 }
+
+func (sv *SegmentVerifier) countPixelParams(params *Params) ([]int64, error) {
```

Comment:
Does this need to be a method on `SegmentVerifier`? Doesn't look like it relies on any `SegmentVerifier` state.

Reply:
it did, but it doesn't any more.  fixed in 06542f7887809487b6899749ab3db683dff211cd